### DISCUSSION TITLE
How to display custom page Action in anywhere.

### DISCUSSION THREAD

**Discussion by valpuia** [filamentphp/filament#6301] [Help] [answered]
If I follow this link [https://filamentphp.com/docs/2.x/admin/pages/actions#getting-started](https://filamentphp.com/docs/2.x/admin/pages/actions#getting-started) I can get Action (button) in custom page.

```
use Filament\Pages\Actions\Action;
 
protected function getActions(): array
{
    return [
        Action::make('settings')->action('openSettingsModal'),
    ];
}
```

I want to rewrite the page heading and I use below line.

```
use Illuminate\Contracts\View\View;

protected function getHeader(): View
{
    return view('filament.headers.kyc-detail', ['name' => $this->name]);
}
```

If I change the heading like this, then I cannot get this actions anymore.. (this is how I want)

But If I want to display this action buttons again on the page, how to called from `blade.php` file

**Comment by danharrin** [comment, marked as answer]
Try `<x-filament::pages.actions :actions="$this->getCachedActions()" class="shrink-0" />`
